src/miner.cpp

```diff
@@ -101,6 +101,9 @@ CBlockTemplate* CreateNewBlock(const CScript& scriptPubKeyIn)
         return NULL;
     CBlock *pblock = &pblocktemplate->block; // pointer for convenience
 
+    /* Initialise the block version.  */
+    pblock->nVersion = CBlockHeader::CURRENT_VERSION;
```

Comment:
See here: https://github.com/dogecoin/dogecoin/blob/master/src/core.h#L403 
That's what my other comment was about.